relays/client-ethereum/Cargo.toml

```diff
@@ -8,12 +8,12 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 [dependencies]
 async-std = "1.6.5"
 bp-eth-poa = { path = "../../primitives/ethereum-poa" }
-codec = { package = "parity-scale-codec", version = "2.0.0" }
+codec = { package = "parity-scale-codec", version = "2.2.0" }
 headers-relay = { path = "../headers" }
 hex-literal = "0.3"
 jsonrpsee-proc-macros = "=0.2.0-alpha.6"
 jsonrpsee-ws-client = "=0.2.0-alpha.6"
 libsecp256k1 = { version = "0.3.4", default-features = false, features = ["hmac"] }
 log = "0.4.11"
 relay-utils = { path = "../utils" }
-web3 = { version = "0.15", git = "https://github.com/tomusdrw/rust-web3", branch ="td-ethabi", default-features = false }
+web3 = { git = "https://github.com/svyatonik/rust-web3.git", branch = "bump-deps" }
```

Comment:
Can you make a PR to the main repo?

Reply:
It requires https://github.com/rust-ethereum/ethabi to be updated && published first. I could open PR there, but this change is SCALE-codec related && I doubt it worth publishing updated ethabi version for ethereum guys just for that. Could start this escalation, though.